src/onINP.ts

```diff
@@ -116,7 +116,7 @@ export const onINP = (onReport: ReportCallback, opts?: ReportOpts) => {
 
   const handleEntries = (entries: Metric['entries']) => {
     (entries as PerformanceEventTiming[]).forEach((entry) => {
-      if (entry.interactionId) {
+      if (entry.entryType == 'first-input' || entry.interactionId) {
```

Comment:
This is only needed until `first-input` gets a proper interactionID.

For now, this leads to the "10 worst" list being 1 larger.  But since web-vitals.js doesn't use that to count number of interactions, it doesn't really matter.  The only time it matters is if the `first-input` duration becomes the 10th largest and ends up evicting another interaction from the list, and then you also have 500 interactions.